web/src/main/java/com/futurewei/alcor/web/entity/elasticip/openstack/FloatingIpRequest.java

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2020 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.web.entity.elasticip.openstack;

```

Comment:
Is this entity used by OpenStack only? If not we could use more generic name.

Reply:
Yes, those two entity is used by Openstack only.